pkg/api/apiloader.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"io/ioutil"
 
+	"github.com/Azure/acs-engine/pkg/agentPoolOnlyApi/v20170831"
```

Comment:
for another PR, but "github.com/Azure/acs-engine/pkg/agentPoolOnlyApi/v20170831", should be under "github.com/Azure/acs-engine/pkg/api/v20170831"